core/tag.h

```diff
@@ -71,6 +71,32 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // Convenience for converting times
 #define BILLION 1000000000LL
 
+/**
+ * Global physical clock offset.
+ * Initially set according to the RTI's clock in federated
+ * programs.
+ */
+extern interval_t _lf_global_physical_clock_offset;
```

Comment:
Would it make sense to call this `_lf_time_physical_offset`? I'm not sure what the `global` part signifies, so I left it out...